### PR TITLE
Setup standalone release

### PR DESCRIPTION
**Pull request by yann-lty**
This PR contains features, fixes and adjustments for the upcoming prebuilt release of Meshroom. 
Related to #128.

### Setup
- [X] initialize executables environment to work with prebuilt package structure (see `setupEnvironment` in `meshroom.__init__`)
- [X] use official PySide2 release from PyPI (excludes Python 2.7 on Windows)
- [X] add dependencies in COPYING file
- [X] make meshroom_photogrammetry an Executable for cx_Freeze
- [X] use Meshroom logo .ico file as application icon on Windows

### UI
- [X] AboutDialog with Open Source Licences from COPYING files info and useful links
- [X] main window footer with cache folder info
- [X] prompt warning when computing an unsaved project
- [X] store main window size in persistent settings

### CODE REVIEW

File: meshroom/ui/qml/main.qml [hunk 1]
```diff
@@ -124,6 +140,22 @@ ApplicationWindow {
...
     }
 
+    MessageDialog {
+        id: unsavedComputeDialog
+
+        canCopy: false
+        icon.text: MaterialIcons.warning
+        preset: "Warning"
+        title: "Unsaved Project"
+        text: "Data will be computed in the default cache folder if project remains unsaved."
```
Comment (fabiencastan) [inline meshroom/ui/qml/main.qml]: Maybe add something like:
"Data will be computed in the default cache folder if project remains unsaved. If you compute and save your project later, the link to your data will be lost."

**Comment by bartoszek** [comment]
This merge introduce new error on the Linux platform (Arch-Linux):
```
copying build/exe.linux-x86_64-3.6/Meshroom -> /home/bartus/AUR/meshroom/pkg/meshroom/usr/lib/Meshroom-2018.1
copying build/exe.linux-x86_64-3.6/meshroom_photogrammetry -> /home/bartus/AUR/meshroom/pkg/meshroom/usr/lib/Meshroom-2018.1
Traceback (most recent call last):
  File "setup.py", line 45, in <module>
    executables=[meshroomExe, meshroomPhotog],
  File "/usr/lib/python3.6/site-packages/cx_Freeze/dist.py", line 349, in setup
    distutils.core.setup(**attrs)
  File "/usr/lib/python3.6/distutils/core.py", line 148, in setup
    dist.run_commands()
  File "/usr/lib/python3.6/distutils/dist.py", line 955, in run_commands
    self.run_command(cmd)
  File "/usr/lib/python3.6/distutils/dist.py", line 974, in run_command
    cmd_obj.run()
  File "/usr/lib/python3.6/distutils/command/install.py", line 557, in run
    self.run_command(cmd_name)
  File "/usr/lib/python3.6/distutils/cmd.py", line 313, in run_command
    self.distribution.run_command(command)
  File "/usr/lib/python3.6/distutils/dist.py", line 974, in run_command
    cmd_obj.run()
  File "/usr/lib/python3.6/site-packages/cx_Freeze/dist.py", line 314, in run
    name = os.path.basename(executable.targetName)
  File "/usr/lib/python3.6/posixpath.py", line 146, in basename
    p = os.fspath(p)
TypeError: expected str, bytes or os.PathLike object, not NoneType
```
[build.error.log](https://github.com/alicevision/meshroom/files/2260502/build.error.log)

**Comment by bartoszek** [comment]
Confirmed, it's working with `targetName` defined.

**Comment by yann-lty** [comment]
Yes I don't see why it's complaining. Anyway, I just made a fallback on our side to define a targetName by default if none is specified !

**Pull request merged**